src/main/java/kirby/Parser.java

```diff
@@ -0,0 +1,53 @@
+package kirby;
+
+import kirby.commands.*;
```

Comment:
Perhaps you could list the individual commands? While it might look cleaner, I believe wildcard imports are not allowed by the coding standard.

Reply:
Ah whenever I change it, Intellij changes it back to *